docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -499,23 +821,23 @@ If you want to access the other services and their UI, you can port-forward thos
 
 ```shell
 Identity:
-> kubectl port-forward svc/camunda-identity 8080:80
+> kubectl port-forward svc/camunda-identity 8069:80 --namespace camunda
 Operate:
-> kubectl port-forward svc/camunda-operate  8081:80
+> kubectl port-forward svc/camunda-operate  8071:80 --namespace camunda
 Tasklist:
-> kubectl port-forward svc/camunda-tasklist 8082:80
+> kubectl port-forward svc/camunda-tasklist 8072:80 --namespace camunda
 Optimize:
-> kubectl port-forward svc/camunda-optimize 8083:80
+> kubectl port-forward svc/camunda-optimize 8073:80 --namespace camunda
 Connectors:
-> kubectl port-forward svc/camunda-connectors 8088:8080
+> kubectl port-forward svc/camunda-connectors 8078:8080 --namespace camunda
 ```
 
 :::note
 Keycloak must be port-forwarded at all times as it is required to authenticate.
 :::
 
 ```shell
-kubectl port-forward services/camunda-keycloak 18080:80
+kubectl port-forward services/camunda-keycloak 8070:80 --namespace camunda
```

Comment:
I'd suggest keeping it aligned with the helm chart.
Otherwise Helm chart is gonna print x and you will say y.
Both will work but still.


```
If you don't have an ingress controller you can use `kubectl port-forward` to access the deployed web application from outside the cluster:

Identity:
> kubectl port-forward svc/camunda-identity 8080:80
Operate:
> kubectl port-forward svc/camunda-operate  8081:80
Tasklist:
> kubectl port-forward svc/camunda-tasklist 8082:80





If you want to use different ports for the services, please adjust the related configs in the values file since these ports are used as redirect URLs for Keycloak.

Authentication via Identity/Keycloak is enabled. To login into one of the services please port-forward to Keycloak
as well, otherwise, a login will not be possible. Make sure you use `18080` as a port.

> kubectl port-forward svc/camunda-keycloak 18080:80
```